### YamlMime:UniversalReference
items:
  - uid: azure-servicefabric.TestErrorChaosEvent
    name: TestErrorChaosEvent
    fullName: TestErrorChaosEvent
    children:
      - azure-servicefabric.TestErrorChaosEvent.kind
      - azure-servicefabric.TestErrorChaosEvent.reason
      - azure-servicefabric.TestErrorChaosEvent.timeStampUtc
    langs:
      - typeScript
    type: interface
    summary: |-
      <span data-ttu-id="f171e-101">Describe un evento de caos que se genera cuando se produce un evento inesperado en el motor de Chaos.</span><span class="sxs-lookup"><span data-stu-id="f171e-101">Describes a Chaos event that gets generated when an unexpected event occurs in the Chaos engine.</span></span>
      <span data-ttu-id="f171e-102">Por ejemplo, debido a la instantánea de clúster que se va a incoherente, mientras que se produzca un error de una entidad, Chaos encontró que la entidad se ha habido un error, lo que sería un evento inesperado.</span><span class="sxs-lookup"><span data-stu-id="f171e-102">For example, due to the cluster snapshot being inconsistent, while faulting an entity, Chaos found that the entity was already faulted -- which would be an unexpected event.</span></span>
    extends:
      name: azure-servicefabric.ChaosEvent
    package: azure-servicefabric
  - uid: azure-servicefabric.TestErrorChaosEvent.kind
    name: kind
    fullName: kind
    children: []
    langs:
      - typeScript
    type: property
    summary: <span data-ttu-id="f171e-103">Discriminador polimórfico</span><span class="sxs-lookup"><span data-stu-id="f171e-103">Polymorphic Discriminator</span></span>
    syntax:
      content: 'kind: string'
      return:
        type:
          - string
    package: azure-servicefabric
  - uid: azure-servicefabric.TestErrorChaosEvent.reason
    name: reason
    fullName: reason
    children: []
    langs:
      - typeScript
    type: property
    summary: '<span data-ttu-id="f171e-104">Describe por qué se ha generado TestErrorChaosEvent.</span><span class="sxs-lookup"><span data-stu-id="f171e-104">Describes why TestErrorChaosEvent was generated.</span></span> <span data-ttu-id="f171e-105">Por ejemplo, Chaos trata de una partición de error, pero encuentra que la partición ya no es tolerante a errores, a continuación, se genera un TestErrorEvent con el motivo que indique que la partición no es tolerante a errores.</span><span class="sxs-lookup"><span data-stu-id="f171e-105">For example, Chaos tries to fault a partition but finds that the partition is no longer fault tolerant, then a TestErrorEvent gets generated with the reason stating that the partition is not fault tolerant.</span></span>'
    optional: true
    syntax:
      content: 'reason?: string'
      return:
        type:
          - string
    package: azure-servicefabric
  - uid: azure-servicefabric.TestErrorChaosEvent.timeStampUtc
    name: timeStampUtc
    fullName: timeStampUtc
    children: []
    langs:
      - typeScript
    type: property
    summary: <span data-ttu-id="f171e-106">La marca de tiempo de UTC cuando se generó este evento de Chaos.</span><span class="sxs-lookup"><span data-stu-id="f171e-106">The UTC timestamp when this Chaos event was generated.</span></span>
    syntax:
      content: 'timeStampUtc: Date'
      return:
        type:
          - Date
    package: azure-servicefabric
references:
  - uid: azure-servicefabric.ChaosEvent
    name: ChaosEvent
    spec.typeScript:
      - name: ChaosEvent
        fullName: ChaosEvent
        uid: azure-servicefabric.ChaosEvent